kong.yml

```diff
@@ -212,4 +213,32 @@ nginx: |
       # Do not remove, additional configuration placeholder for some plugins
       # {{additional_configuration}}
     }
+
+    server {
+      listen {{consumer_api_port}};
+
+      location / {
+        default_type application/json;
+        content_by_lua '
+          ngx.header["Access-Control-Allow-Origin"] = "*"
```

Comment:
I can change nginx conf so it will be like:
<pre>
    server {
      listen {{admin_api_port}};

      location / {
        default_type application/json;
        access_by_lua_file 'path_to/api_access.lua'
        content_by_lua '
          local lapis = require "lapis"
          lapis.serve("kong.api.app")
        ';
      }

      include includes/api_locations.conf;
      # Do not remove, additional configuration placeholder for some plugins
      # {{additional_configuration}}
    }

    server {
      listen {{consumer_api_port}};

      location / {
        default_type application/json;
        access_by_lua_file 'path_to/api_access.lua'
        content_by_lua '
          local lapis = require "lapis"
          lapis.serve("kong.consumer_api.app")
        ';
      }
      include includes/api_locations.conf;
    }
</pre>
api_access.lua:
<pre>
          ngx.header["Access-Control-Allow-Origin"] = "*"
          if ngx.req.get_method() == "OPTIONS" then
            ngx.header["Access-Control-Allow-Methods"] = "GET,HEAD,PUT,PATCH,POST,DELETE"
            ngx.exit(204)
          end
</pre>

and include:
<pre>
      location /nginx_status {
        internal;
        stub_status;
      }

      location /robots.txt {
        return 200 'User-agent: *\nDisallow: /';
      }

</pre>

and includes/api_locations.conf;
<pre>
      location /nginx_status {
        internal;
        stub_status;
      }

      location /robots.txt {
        return 200 'User-agent: *\nDisallow: /';
      }

</pre>

how about this ?

I Also thought to use content_by_lua_file file lapis, but there is an open issue about memory leak - https://github.com/leafo/lapis/issues/203.